### PR TITLE
Simple UI i18n consistency fixes detected after completing translation

### PR DESCRIPTION
**Pull request by wget**
#### Summary
* Simply be consistent when using Ex vs E.g. or eg. (cf. [this article](http://painintheenglish.com/case/4723) and [this stackoverflow answer](http://english.stackexchange.com/a/28173) (**accepted**)
* A change brought by @jasonblais. [Is the absence of capital letter to `Documentation` intended](https://github.com/mattermost/platform/pull/4034/commits/bbbeece23c01a747341605942375abe1a47febaa#diff-48187f065420cda8aa0582d6c6cc4b2cR1367) ? (**removed** as it was intended, see below)
* Remove unneeded spaces from UI. (**removed** see below)

#### Ticket Link
N/A

#### Checklist
- [x] Has UI changes
- [x] Includes text changes and localization files updated

### CODE REVIEW

File: webapp/i18n/en.json [hunk 1]
```diff
@@ -1249,7 +1249,7 @@
   "help.attaching.limitations": "## File Size Limitations\nMattermost supports a maximum of five attached files per post, each with a maximum file size of 50Mb.",
   "help.attaching.methods": "## Attachment Methods\nAttach a file by drag and drop or by clicking the attachment icon in the message input box.",
   "help.attaching.notSupported": "Document preview (Word, Excel, PPT) is not yet supported.",
-  "help.attaching.pasting": "#### Pasting Images\nOn Chrome and Edge browsers, it is also possible to upload files by pasting them from the clipboard. This is not yet supported on other browsers.",
+  "help.attaching.pasting": "#### Pasting Images\nOn Chrome and e.g. browsers, it is also possible to upload files by pasting them from the clipboard. This is not yet supported on other browsers.",
```
Comment (jasonblais) [inline webapp/i18n/en.json]: This one actually refers to the 'Edge' browser, so is correct

**Comment by jasonblais** [comment]
Many thanks @wget! 

1. I'll queue the use of 'ex' v.s. 'e.g.' to our UX meeting to confirm
2. The uncapitalized version of `documentation` is actually intended here.. It appears as the placeholder for `documentation` in the attached image
3. Thanks for removing the unneeded spaces :)

image

**Comment by wget** [comment]
@jasonblais Ok. Integrated your remarks. Edited my commits.

**Comment by enahum** [comment]
@wget this PR needs a rebase :)

**Comment by jasonblais** [comment]
Hi @wget, and many thanks again for the pull request!

We reviewed a few of the strings on the testing server, and the space is needed in some cases so that they appear properly on the user interface. Sorry for initially overlooking this.

We would either need to individually check whether the string needs the additional space, or revert back to what we have now. I think the latter might be easier as checking each of the ~20-30 strings with an extra space may not be worth the time.

What do you think?

We'd still love to take the "Ex --> E.g." changes, our UX team confirmed that is more standard.

**Comment by wget** [comment]
@enahum Rebased :-)

**Pull request merged**